### PR TITLE
feat? added contracts for analytics

### PR DESCRIPTION
**Pull request by CaelanHill**

### CODE REVIEW

File: docs/docs/dev-docs/api-contracts/Analytics.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+}
+```
+```json
+ //idk what the ts of this is
+ // This is a GroupedValue format
+ {
+    "key" : number,
+    ...
+ }
+```
```
Comment (ghost) [inline docs/docs/dev-docs/api-contracts/Analytics.md:19]: are you talking about this?
```ts
interface MyType {
  [key: string]: number
}
```

```

File: docs/docs/dev-docs/api-contracts/Analytics.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+//returns
+//if resolved disputes exist
+interface ResponseResolvedDisputes {
+    data : time
+}
+//otherwise
+interface error {
+    error : string
+}
+```
```
Comment (ghost) [inline docs/docs/dev-docs/api-contracts/Analytics.md:38]: very small nit, but you can just say
```ts
type ResponseResolvedDisputes = number;
```
We assume the result type for everything

File: docs/docs/dev-docs/api-contracts/Analytics.md (hunk 1 above)
Comment (CaelanHill) [reply docs/docs/dev-docs/api-contracts/Analytics.md:19]: yes

File: docs/docs/dev-docs/api-contracts/Analytics.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+# Aggregate analytics
+
+- **Endpoint:** `POST /stats/{table}`
+- **Headers:**
+  - `Authorization: Bearer <JWT>`
+- **Note:** only available to admins
+
+```ts 
+//queries the count of records, applying the below params if supplied
+interface request{
```
Comment (ghost) [inline docs/docs/dev-docs/api-contracts/Analytics.md:63]: lowercase :(

**Review by ghost — APPROVED** [review]
small things, but lgtm :) Thank you bubz

**Pull request merged**